client/session.go

```diff
@@ -33,6 +33,9 @@ import (
 	"sync/atomic"
 	"time"
 
+	"github.com/uber-go/tally"
+	"github.com/uber/tchannel-go/thrift"
```

Comment:
Any reason to put these above internal imports?  This breaks the pattern in use in other places.